actix_todo/templates/index.html.tera

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Actix Todo Example</title>
+
+    <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="/static/css/normalize.css">
+    <link rel="stylesheet" href="/static/css/skeleton.css">
+    <link rel="stylesheet" href="/static/css/style.css">
+</head>
+<body>
+  <div class="container">
+    <p><!-- nothing to see here --></p>
+
+    <div class="row">
+      <h4>Actix Todo</h4>
+      <form action="/todo" method="post">
+        <div class="ten columns">
+          <input type="text" placeholder="enter a task description ..."
```

Comment:
Well spotted. Yes, ordinarily this would be a great addition. Except if we do this here, the validation would be done in the browser instead of in our handlers, and so our nice error flash message ...

<img width="276" alt="image" src="https://user-images.githubusercontent.com/306583/44811302-3f640e00-abcc-11e8-8485-745e570ba1da.png">

... would never appear

Reply:
Oh yeah... I guess it is good example to show such sort of validation. Ok then I wasn't reading your intention carefully here 😄